### PR TITLE
docs: create README for lwc package

### PR DESCRIPTION
**Pull request by jbleyleSF**
## Details
Created a README for the lwc package so we can publish it to https://www.npmjs.com/package/lwc

## Does this PR introduce breaking changes?

* ✅ `No, it does not introduce breaking changes.`

If yes, please describe the impact and migration path for existing applications.

## The PR fulfills these requirements:
* Have tests for the proposed changes been added? N/A
* Have you followed [these instructions](../CONTRIBUTING.md#-commit-message-conventions) to clearly describe the issue being fixed or feature enhanced? ✅ 

## GUS work item
W-7408095

### CODE REVIEW

File: packages/lwc/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+Lightning Web Components (LWC) is a blazing fast, enterprise-grade web components foundation for building user interfaces. LWC is a thin layer around the low-level Web Components APIs. The `lwc` package is the main entry point for dependencies.
```
Comment (Gr8Gatsby) [inline packages/lwc/README.md:1]: How about: LWC provides a simple authoring format for UI Components. LWC compiles this authoring format into low-level Web Component APIs.

Comment (Gr8Gatsby) [reply packages/lwc/README.md:1]: This is for the Second Sentence.

Comment (jbleyleSF) [inline packages/lwc/README.md:1]:
```suggestion
Lightning Web Components (LWC) is a blazing fast, enterprise-grade web components foundation for building user interfaces. LWC provides a simple authoring format for UI Components. LWC compiles this authoring format into low-level Web Component APIs. The `lwc` package is the main entry point for dependencies.
```

File: packages/lwc/README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+Lightning Web Components (LWC) is a blazing fast, enterprise-grade web components foundation for building user interfaces. LWC provides a simple authoring format for UI Components. LWC compiles this authoring format into low-level Web Component APIs. The `lwc` package is the main entry point for dependencies.
+
+-   Develop components quickly and declaratively using HTML, JavaScript, and CSS.
```
Comment (caridy) [inline packages/lwc/README.md:3]: one feedback that I have received multiple times is that the statically analyzable aspect of the framework, as a foundational part, is not mentioned anywhere. It only make reference to the declarative aspect of it... but I think we should mentioned somewhere that part of the deal is to author components in a way that the code can be statically analyzed .

File: packages/lwc/README.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
+Lightning Web Components (LWC) is a blazing fast, enterprise-grade web components foundation for building user interfaces. LWC provides a simple authoring format for UI Components. LWC compiles this authoring format into low-level Web Component APIs. The `lwc` package is the main entry point for dependencies.
```
Comment (kevinv11n) [inline packages/lwc/README.md:1]: Re blazing fast, we provide no data justifying this claim.

Comment (Gr8Gatsby) [reply packages/lwc/README.md:1]: Let's focus on enterprise-grade only. I think we drop blazing-fast unless we quantify.

Comment (diervo) [reply packages/lwc/README.md:1]: +1 we did that when we launch to differenciate from aura mostly. I think now   the message is not needed

Comment (jbleyleSF) [inline packages/lwc/README.md:1]:
```suggestion
Lightning Web Components (LWC) is an enterprise-grade web components foundation for building user interfaces. LWC provides a simple authoring format for UI Components. LWC compiles this authoring format into low-level Web Component APIs. The `lwc` package is the main entry point for dependencies.
```

File: packages/lwc/README.md (hunk 2 above)
Comment (Gr8Gatsby) [reply packages/lwc/README.md:3]: I think we can develop this out a bit for an update of the readme later on.

**Pull request merged**